From: Josep Puigdemont <josep.puigdemont@enea.com>
Date: Tue, 17 May 2016 14:40:48 +0200
Subject: [PATCH] deploy.py: add a time stamp to the new ISO image

Add a time stamp to the newly create ISO image. This is to prevent name
collisions with other ISO images from earlier deploy jobs.

Ideally this will only be a temporary need, and releng will provide with
unique IDs for each job that we can use to this effect.

Signed-off-by: Josep Puigdemont <josep.puigdemont@enea.com>
---
 deploy/deploy.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/deploy/deploy.py b/deploy/deploy.py
index 041ba2f..cebef97 100755
--- a/deploy/deploy.py
+++ b/deploy/deploy.py
@@ -13,6 +13,7 @@ import os
 import io
 import re
 import sys
+import time
 import yaml
 import errno
 import signal
@@ -100,8 +101,9 @@ class AutoDeploy(object):
 
     def install_fuel_master(self):
         log('Install Fuel Master')
-        new_iso = ('%s/deploy-%s'
-                   % (self.tmp_dir, os.path.basename(self.iso_file)))
+        stamp = time.strftime("%Y%m%d%H%M%S")
+        new_iso = ('%s/deploy-%s-%s'
+                   % (self.tmp_dir, stamp, os.path.basename(self.iso_file)))
         self.patch_iso(new_iso)
         self.iso_file = new_iso
         self.install_iso()
